Polygon_mesh_processing/doc/Polygon_mesh_processing/NamedParameters.txt

```diff
@@ -152,6 +152,30 @@ the property map containing information about edges of the input polygon mesh be
 a default property map where no edge is constrained is provided.
 \cgalNPEnd
 
+\cgalNPBegin{ maximum_number_of_patches } \anchor PMP_maximum_number_of_patches
+contains the surface patch beeing treated +1.\n
+\b Type : `std::size_t`* \n
```

Comment:
In the begin, there was a functor with a member variable that didn't seem to be used outside of one of the functions. As I removed the functor, I realized that this member variable was used in a particular mesh domain, so I introduced a named parameter to replace it. This named parameter needs to keep the last value that was given to it so the complex mesh domain can keep attributing patches to several polyhedra at once. This is why I need it to be a pointer, as I cannot do it with a reference, mainly because the return value of choose_param() is unknown to me, and probably a temporary variable.